Enterprise Bot Template points to non-existent i18n resource

**Project**
Enterprise Bot Template

**Description**
When starting a new bot created with the yeoman template, it crashes when the Emulator connects:

```js
{ Error: Cannot find module './resources/Intro.json'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
    at Function.Module._load (internal/modules/cjs/loader.js:506:25)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at Module.patchedRequire [as require] (/my-bot/node_modules/diagnostic-channel/dist/src/patchRequire.js:14:46)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Function.buildIntroCard (/my-bot/lib/dialogs/main/mainResponses.js:18:27)
    at MainResponses.responseTemplates.Map (/my-bot/lib/dialogs/main/mainResponses.js:95:50)
    at DictionaryRenderer.renderTemplate (/my-bot/lib/dialogs/templateManager/dictionaryRenderer.js:28:32)
    at MainResponses.renderTemplate (/my-bot/lib/dialogs/templateManager/templateManager.js:60:55)
    at MainResponses.replyWith (/my-bot/lib/dialogs/templateManager/templateManager.js:41:42) code: 'MODULE_NOT_FOUND' }
```

**To Reproduce**
1. Create a new enterprise bot using yeoman
1. Create the required services using msbot
1. Configure the .env.development
1. npm start
1. Open the .bot file using BF Emulator

**Expected behavior**
The Emulator connects successfully

**Additional context**
I poked around the en.json config, and it includes the property `"introPath": "./resources/Intro.json"`, however this file does not exist in the template

Template was creating from yeoman generator associated with git branch `master 5f423895`

--- a/solutions/Virtual-Assistant/src/typescript/assistant/package.json
+++ b/solutions/Virtual-Assistant/src/typescript/assistant/package.json
@@ -7,7 +7,7 @@
     "main": "lib/index.js",
     "scripts": {
         "clean": "rimraf ./lib",
-        "copy-templates": "copyfiles --up 1 ./src/**/*.json ./lib",
+        "copy-templates": "copyfiles --up 1 \"./src/**/*.json\" \"./lib\"",
         "prebuild": "npm run lint",
         "build": "tsc --p tsconfig.json && npm run copy-templates",
         "post-install": "npm run build && node ./deploymentScripts/webConfigPrep.js",

--- a/templates/Enterprise-Template/src/typescript/enterprise-bot/package.json
+++ b/templates/Enterprise-Template/src/typescript/enterprise-bot/package.json
@@ -7,7 +7,7 @@
     "main": "lib/index.js",
     "scripts": {
         "clean": "rimraf ./lib",
-        "copy-templates": "copyfiles --up 1 ./src/**/*.json ./lib",
+        "copy-templates": "copyfiles --up 1 \"./src/**/*.json\" \"./lib\"",
         "prebuild": "npm run lint",
         "build": "tsc --p tsconfig.json && npm run copy-templates",
         "postinstall": "npm run build && node ./deploymentScripts/webConfigPrep.js",

--- a/templates/Enterprise-Template/src/typescript/generator-botbuilder-enterprise/generators/app/templates/enterprise-bot/_package.json
+++ b/templates/Enterprise-Template/src/typescript/generator-botbuilder-enterprise/generators/app/templates/enterprise-bot/_package.json
@@ -7,7 +7,7 @@
     "main": "lib/index.js",
     "scripts": {
         "clean": "rimraf ./lib",
-        "copy-templates": "copyfiles --up 1 ./src/**/*.json ./lib",
+        "copy-templates": "copyfiles --up 1 \"./src/**/*.json\" \"./lib\"",
         "prebuild": "npm run lint",
         "build": "tsc --p tsconfig.json && npm run copy-templates",
         "postinstall": "npm run build && node ./deploymentScripts/webConfigPrep.js",

